packages/wallets/src/routes/WalletsListingRoute/WalletsListingRoute.tsx

```diff
@@ -53,10 +53,15 @@ const WalletsListingRoute: React.FC = () => {
     }, [platformMapping, resetTradingPlatformActionParams, show]);
 
     useEffect(() => {
-        if (!activeWallet && firstLoginid) {
-            switchAccount(firstLoginid);
+        if (!activeWallet) {
+            const activeLinkedAccount = walletAccounts?.find(wallet => wallet.is_linked_account_active);
```

Comment:
Question @rostislav-deriv currently we have [this bug](https://app.clickup.com/t/20696747/WALL-3498) where on initial login we are not getting redirected to the fiat account and instead to the first account in the array before it gets sorted. 

By linking the active wallet account to the trading account would this solve the issue and set the correct initial account?

Reply:
also side note (outside of scope for this PR): the current flow is to: load account, then switch account in wallets. While ideally, we should load correct / default account to begin with (just asking @nijil-deriv Melvinder to create / add separate card for this refactor)